### PR TITLE
RITS-53 - RITS-53-siirra-kaikki-scriptit-scripts-kansioon

### PR DESCRIPTION
**Pull request by kymppi**

### CODE REVIEW

File: scripts/remove-docker.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+function removeNetwork () {
+    docker network remove $1 >> /dev/null;
+    echo "Removed network $1" 
+}
+
+for i in "services" "rmq" "grafana" "databases"; do
+  echo "Removing $i..."
+  docker-compose -f ./docker/$i/docker-compose.yml down
```
Comment (raikasdev) [inline scripts/remove-docker.sh:9]: Shouldn't this be ../ as the docker directory is in the root directory?

File: scripts/start-docker.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+for i in "grafana" "rmq" "databases"; do
+    echo "Creating network ritta-$i"
+    createNetwork "ritta-$i"
+done
+
+for i in "rmq" "grafana" "databases" "services"; do
+  echo "Starting $i..."
+
+  if [ "$i" = "services" ]; then
+    docker-compose -f ./docker/$i/docker-compose.yml up
```
Comment (raikasdev) [inline scripts/start-docker.sh:18]: ../

File: scripts/start-docker.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    createNetwork "ritta-$i"
+done
+
+for i in "rmq" "grafana" "databases" "services"; do
+  echo "Starting $i..."
+
+  if [ "$i" = "services" ]; then
+    docker-compose -f ./docker/$i/docker-compose.yml up 
+  else
+    docker-compose -f ./docker/$i/docker-compose.yml up -d &
```
Comment (raikasdev) [inline scripts/start-docker.sh:20]: ../

File: scripts/stop-docker.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+for i in "services" "rmq" "grafana" "databases"; do
+  echo "Stopping $i..."
+  docker-compose -f ./docker/$i/docker-compose.yml stop
```
Comment (raikasdev) [inline scripts/stop-docker.sh:4]: ../

**Review by raikasdev — CHANGES_REQUESTED** [review]

**Comment by kymppi** [comment]
No, it takes the path from the directory where you execute the scripts, for example if you're in the project root and run `./scripts/start-docker.sh` it successfully gets the right docker-compose etc.

**Review by raikasdev — APPROVED** [review]

**Pull request merged**